src/clients/MultiCallerClient.ts

```diff
@@ -183,12 +183,21 @@ export class MultiCallerClient {
     );
     const batchSimResults = await this.txnClient.simulate(batchTxns);
     const batchesAllSucceeded = batchSimResults.every(({ succeed, transaction, reason }, idx) => {
-      this.logger[succeed ? "debug" : "error"]({
-        at: "MultiCallerClient#executeChainTxnQueue",
-        message: `${succeed ? "Successfully simulated" : "Failed to simulate"} ${networkName} transaction batch!`,
-        batchTxn: { ...transaction, contract: transaction.contract.address },
-        reason,
-      });
+      if (this.canIgnoreRevertReason({ succeed, transaction, reason })) {
+        this.logger.debug({
+          at: "MultiCallerClient#executeChainTxnQueue",
+          message: `Ignoring revert reason for ${networkName} transaction batch!`,
+          batchTxn: { ...transaction, contract: transaction.contract.address },
+          reason,
+        });
+      } else {
+        this.logger[succeed ? "debug" : "error"]({
+          at: "MultiCallerClient#executeChainTxnQueue",
+          message: `${succeed ? "Successfully simulated" : "Failed to simulate"} ${networkName} transaction batch!`,
+          batchTxn: { ...transaction, contract: transaction.contract.address },
+          reason,
+        });
+      }
```

Comment:
How about this? The only difference is the log level of the emitted message, rather than the message being different. But I think that's OK (??)

```suggestion
        this.logger[succeed || this.canIgnoreRevertReason({ succeed, transaction, reason }) ? "debug" : "error"]({
          at: "MultiCallerClient#executeChainTxnQueue",
          message: `${succeed ? "Successfully simulated" : "Failed to simulate"} ${networkName} transaction batch!`,
          batchTxn: { ...transaction, contract: transaction.contract.address },
          reason,
        });
      }
```

Reply:
Oh nice great refactor